policy/release/lib/attestations.rego

```diff
@@ -47,12 +51,36 @@ results_named(name) = results {
 		result_map := json.unmarshal(result.value)
 
 		# Inject the task name so we can show it in failure messages
-		r := object.union(result_map, {task_name: task.name})
+		r := object.union(result_map, task_data(task))
 	]
 }
 
+# Returns the data relating to the task if the task is referenced from a bundle
+task_data(task) = info {
+	task.ref
+	task.ref.kind == "Task"
+	info := {key_task_name: task.ref.name, key_bundle: task.ref.bundle}
+}
+
+# Returns the data relating to the task if the task is not referenced from a
+# bundle
+task_data(task) = info {
+	not task.ref
+	not task.ref.kind == "Task"
+	info := {key_task_name: task.name}
+}
+
 # (Don't call it test_results since test_ means a unit test)
-results_from_tests := results_named(hacbs_test_task_result_name)
+results_from_tests = results {
```

Comment:
Is there a reason we can't just move this logic to `results_named`? Might as well make that function safer. Maybe a future PR.